lib/plugins/processor/asset.js

```diff
@@ -33,6 +33,9 @@ module.exports = ctx => {
       file.read()
     ]).spread((stats, content) => {
       const data = yfm(content);
+      if (data.private) {
```

Comment:
`data.publish === false` as we discussed in https://github.com/hexojs/hexo/issues/4433#issuecomment-679838640.